### PR TITLE
fix(ui): supported tooltip in user description

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes:

- https://github.com/open-metadata/OpenMetadata/issues/13316#issuecomment-1760124069

- supported tooltip in user description
- add generic support to add tooltip in rich-text-editor

image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx [hunk 1]
```diff
@@ -95,17 +96,35 @@ const RichTextEditorPreviewer = ({
...
-        data-testid="markdown-parser">
-        <Viewer
-          extendedAutolinks
-          customHTMLRenderer={customHTMLRenderer}
-          initialValue={viewerValue}
-          key={uniqueId()}
-          linkAttributes={{ target: '_blank' }}
-        />
-      </div>
+      <Popover
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx:99]: If no tooltip then we should not be rendering the popover itself.

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx]: @Sachin-chaurasiya  updated the changes as per comments!

**Comment by chirag-madlani** [comment]
@Ashish8689 I believe we should go with an approach that we took for the Teams page instead of tooltip. 
cc: @harshach @Rupchavan

**Comment by Ashish8689** [comment]
Closing this PR, as we have revamped the profile page that solves description issue https://github.com/open-metadata/OpenMetadata/pull/13744

**Pull request closed without merging**